### ISSUE TITLE
Set PHP to version 7

### ISSUE THREAD

**Issue by jenlampton** [backdrop-ops/backdrop-pantheon#9]
Since we will shortly have a `pantheon.yml` file I recommend we set the PHP version to PHP 7 for new Backdrop sites on Pantheon. 

One note: I added a this setting to an existing Backdrop site and it upgraded the pantheon instance to PHP 7. I'm not sure what ramifications that may have for existing Backdrop sites on Pantheon if we blanket updated them all, but my site was fine.

**Comment by herbdool** [comment]
@serundeputy @jenlampton Pantheon has now pushed a pantheon.upstream.yml file to each of its upstream projects (for Drupal 7, 8 and Wordpress) that looks like:

```
# IMPORTANT NOTE:
# Do not edit this file unless you are doing so in your custom upstream repository.
# Override the defaults specified here in a site-specific `pantheon.yml` file.
# For more information see: https://pantheon.io/docs/pantheon-upstream-yml
api_version: 1
php_version: 7.0
```
I think we should have this *plus* set the drush version so there's a chance that we can have it working with Backdrop:

```
drush_version: 8
```

I think this is the preferred approach now and people can create their own pantheon.yml file if they *don't* want to be on php_version: 7.0 or if they want to tweak other settings.

**Comment by herbdool** [comment]
The other part is writing a commit message that gives them more info:

![selection_006](https://user-images.githubusercontent.com/1071483/28121336-b9a0c69c-66e8-11e7-9fd2-d39d5dd8dbe7.png)

**Comment by jenlampton** [comment]
Thanks for the follow-up @herbdool, great ideas, both :)

**Comment by kreynen** [comment]
We applied this PR to the custom upstream we're using for Backdrop sites and it is working well.

**Comment by herbdool** [comment]
I think this should be in  pantheon.upstream.yml and then people can override in pantheon.yml

**Comment by herbdool** [comment]
As per @greg-1-anderson recommendations, I've got a [PR](https://github.com/backdrop-ops/backdrop-pantheon/pull/23) that creates pantheon.upstream.yml and sets it to PHP 7.1 there. People can still override it with their own pantheon.yml in their own site.

**Comment by kreynen** [comment]
This should be closed.

**Comment by jenlampton** [comment]
Looks like https://github.com/backdrop-ops/backdrop-pantheon/pull/23 hasn't been merged yet. Am I missing something?

**Comment by herbdool** [comment]
Correct. #23 can still be merged but this one can be closed since it's not the right file.

**Comment by kreynen** [comment]
This actually conflicts with https://github.com/backdrop-ops/backdrop-pantheon/pull/23 and should not be merged.  The pantheon.upstream.yml version solves multiple problems and is the solution @greg-1-anderson recommends.  Leaving this PR open for 2 years is just going to confuse the next person.

**Comment by serundeputy** [comment]
closed in favour of #23
